src/class-parsely.php

```diff
@@ -92,6 +93,24 @@ public function __construct() {
 		add_action( 'wp_enqueue_scripts', [ $this, 'wp_parsely_style_init' ] );
 	}
 
+	function add_parsely_link( $actions, $post ) {
```

Comment:
```suggestion
	public function add_parsely_link( $actions, $post ) {
```

Also needs a DocBlock.